From da748e9eda149e69e859ac57f4f6072b6e3a64f6 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Thu, 19 Mar 2020 09:02:38 +0700
Subject: [PATCH 172/441] dt-bindings: arm: renesas: Add HopeRun RZ/G2H boards

This patch adds board HiHope RZ/G2H (the main board, powered by
the R8A774E1) and board HiHope RZ/G2 EX (the expansion board
that sits on top of the HiHope RZ/G2H). Both boards are made
by Jiangsu HopeRun Software Co., Ltd. (a.k.a. HopeRun).

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 Documentation/devicetree/bindings/arm/shmobile.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/shmobile.txt b/Documentation/devicetree/bindings/arm/shmobile.txt
index 58a9c51..51b8fff 100644
--- a/Documentation/devicetree/bindings/arm/shmobile.txt
+++ b/Documentation/devicetree/bindings/arm/shmobile.txt
@@ -96,11 +96,14 @@ Boards:
   - HopeRun expansion board for HiHope RZ/G2 platforms
     compatible = "hoperun,hihope-rzg2m", "renesas,r8a774a1",
                  "hoperun,hihope-rzg2n", "renesas,r8a774b1",
+                 "hoperun,hihope-rzg2h", "renesas,r8a774e1",
                  "hoperun,hihope-rzg2-ex"
   - HopeRun HiHope RZ/G2M platform
     compatible = "hoperun,hihope-rzg2m", "renesas,r8a774a1"
   - HopeRun HiHope RZ/G2N platform
     compatible = "hoperun,hihope-rzg2n", "renesas,r8a774b1"
+  - HopeRun HiHope RZ/G2H platform
+    compatible = "hoperun,hihope-rzg2h", "renesas,r8a774e1"
   - iWave Systems RZ/G1C Single Board Computer (iW-RainboW-G23S)
     compatible = "iwave,g23s", "renesas,r8a77470"
   - iWave Systems RZ/G1E SODIMM SOM Development Platform (iW-RainboW-G22D)
-- 
2.7.4

